src/services/confirmCompanyService.ts

```diff
@@ -0,0 +1,56 @@
+import { lookupCompanyStatus, lookupCompanyType } from "../lib/utils/api_enumerations";
+import { CompanyProfile } from "@companieshouse/api-sdk-node/dist/services/company-profile/types";
+import { toReadableFormat } from "../lib/utils/date";
+
+export const formatForDisplay = (companyProfile: CompanyProfile) => {
+    const registeredOfficeAddress = {
+        addressLineOne: formatTitleCase(companyProfile.registeredOfficeAddress.addressLineOne),
+        addressLineTwo: formatTitleCase(companyProfile.registeredOfficeAddress.addressLineTwo),
+        locality: formatTitleCase(companyProfile.registeredOfficeAddress.locality),
+        region: formatTitleCase(companyProfile.registeredOfficeAddress.region),
+        country: formatTitleCase(companyProfile.registeredOfficeAddress.country),
+        postalCode: companyProfile.registeredOfficeAddress.postalCode ? companyProfile.registeredOfficeAddress.postalCode.toUpperCase() : null,
+        poBox: companyProfile.registeredOfficeAddress.poBox ? companyProfile.registeredOfficeAddress.poBox.toUpperCase() : null
+    };
+
+    return {
+        companyNumber: companyProfile.companyNumber,
+        companyName: companyProfile.companyName,
+        type: lookupCompanyType(companyProfile.type),
+        companyStatus: lookupCompanyStatus(companyProfile.companyStatus),
+        dateOfCreation: toReadableFormat(companyProfile.dateOfCreation),
+        registeredOfficeAddress: registeredOfficeAddress
+    };
+};
+
+export const formatTitleCase = (str: string|undefined): string => {
+    if (!str) {
+        return "";
```

Comment:
Test coverage indicates that this is not tested

Reply:
This is just returning an empty string if no string is passed, we don't need 100% test code coverage. I've tested the formatting of the api responses in the router test but I can't check if its returning an empty string to the page.